sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -263,8 +273,8 @@ object GpuCast {
         withResource(input.min()) { minInput =>
           withResource(input.max()) { maxInput =>
             if (minInput.isValid && minInput.getBigDecimal().compareTo(min) == -1 ||
-                maxInput.isValid && maxInput.getBigDecimal().compareTo(max) == 1) {
-              throw new ArithmeticException(GpuCast.OVERFLOW_MESSAGE)
+              maxInput.isValid && maxInput.getBigDecimal().compareTo(max) == 1) {
```

Comment:
The original indentation was clearer to me, to indicate continuation of the expression from the previous line.